apps/ui/src/networks/offchain/api/types.ts

```diff
@@ -52,7 +52,7 @@ export type ApiSpace = {
     type: VoteType | '' | null;
     quorum: number | null;
     quorumType?: 'default' | 'rejection';
-    privacy: string;
+    privacy: '' | 'shutter' | 'any';
```

Comment:
Shouldn't that empty string be `none` instead ? Since we fallback the formatSpace with `space.voting.privacy || 'none'`